samples/app-sso/csharp/App SSO Sample/AdapterWithErrorHandler.cs

```diff
@@ -5,15 +5,16 @@
 using Microsoft.Bot.Builder;
 using Microsoft.Bot.Builder.Integration.AspNet.Core;
 using Microsoft.Bot.Builder.TraceExtensions;
+using Microsoft.Bot.Connector.Authentication;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
 
 namespace Microsoft.BotBuilderSamples
 {
-    public class AdapterWithErrorHandler : BotFrameworkHttpAdapter
+    public class AdapterWithErrorHandler : CloudAdapter
```

Comment:
Do we need to inherit it from Cloud ?
It could be a big change for lots of samples?

Reply:
Yes it is deprecated. As of latest package